### PR TITLE
Update deps and split auth credential for Task and rest

### PR DESCRIPTION
**Pull request by stevenvegt**
Make oapi-codegen commands compatible with v11.x
Update the nuts client code to latest sha
Create separate auth credential for the Task
If the task is completed, don't try to show patient information etc.

### CODE REVIEW

**Comment by reinkrul** [comment]
Why issue 2 VCs? If important, should we inform the community this is best practice?

File: web/src/ehr/inbox/TransferRequest.vue [hunk 1]
```diff
@@ -32,7 +32,7 @@
           </div>
         </div>
 
-        <div class="p-5">
+        <div class="p-5" v-if="transferRequest.status != 'completed'">
```
Comment (reinkrul) [inline web/src/ehr/inbox/TransferRequest.vue:35]: Why not show this when completed?

File: domain/transfer/sender/service.go [hunk 2]
```diff
@@ -718,6 +709,37 @@ func (s service) AssignTransfer(ctx context.Context, customerID int, transferID,
...
+	// Create an Auth Credential for the Task
+	authorizedTask := s.taskForNursingHandoff(transferTask.ID)
+	if err := s.vcr.CreateAuthorizationCredential(ctx, customerDID, &credential.NutsAuthorizationCredentialSubject{
+		ID: organizationDID,
+		LegalBase: credential.LegalBase{
+			ConsentType: "implied",
+		},
+		PurposeOfUse: transfer.SenderServiceName,
+		Resources:    authorizedTask,
+	}); err != nil {
```
Comment (pascaldekloe) [inline domain/transfer/sender/service.go:723]: The `if` initializer is not well suited for multiline constructions. 😄 Kudos for how easy this methods reads though.

**Review by pascaldekloe — APPROVED** [review]

File: web/src/ehr/inbox/TransferRequest.vue (hunk 1 above)
Comment (stevenvegt) [reply web/src/ehr/inbox/TransferRequest.vue:35]: Data is no longer shared when completed

**Comment by stevenvegt** [comment]
> Why issue 2 VCs? If important, should we inform the community this is best practice?

Yes, Wout is adding this to the next meeting, this is also already tested in the last hackathon.

**Pull request merged**